frontend/src/components/user/UserAvatar.tsx

```
@@ -0,0 +1,103 @@
+            />
+            <AvatarFallback>{currentUser.firstName.charAt(0)}</AvatarFallback>
+          </Avatar>
+        </Link>
+      </HoverCardTrigger>
+      <HoverCardContent
+        align="end"
+        className="w-64 bg-background/60 shadow-md rounded-lg p-3 z-50"
+      >
+        <div className="flex flex-col items-center">
+          <Avatar className="w-12 h-12 border border-gray-300">
+            <AvatarImage src={currentUser.avatar!!} />
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)